infrastructure/docker-compose.app.yml

```
@@ -51,6 +51,7 @@ services:
       - CERT_PUBLIC_KEY_PATH=/run/secrets/jwt-public-key.{{STACK}}.{{ts}}
       - MONGO_URL=mongodb://${STACK}__notification:${NOTIFICATION_MONGODB_PASSWORD}@mongo1/${STACK}__notification?replicaSet=rs0
       - COUNTRY_CONFIG_URL=http://countryconfig.{{STACK}}_app_net:3040
+      - USER_MANAGEMENT_URL=http://user-mgnt:3030/
```

```suggestion
            - USER_MANAGEMENT_URL=http://user-mgnt.{{STACK}}_app_net:3030/
```
The URLs here are a bit different from our usual ones :sweat_smile: 
For context, this is due to the e2e server running multiple deployments as opposed to our usual one server = one deployment